sr_robot_commander/test/test_robot_commander.py

```diff
@@ -0,0 +1,465 @@
+        self.assertTrue(condition_1 and condition_2)
+
+    def test_set_max_velocity_scaling_factor__range_ok(self):
+        raised = False
+        try:
+            self.robot_commander.set_max_velocity_scaling_factor(0.2)
+        except Exception:
+            raised = True
+        self.assertFalse(raised)
+
+    def test_set_max_velocity_scaling_factor__range_not_ok(self):
+        raised = False
+        try:
+            self.robot_commander.set_max_velocity_scaling_factor(3)
+        except Exception:
+            raised = True
+        self.assertTrue(raised)
+
+    def test_set_max_acceleration_scaling_factor__range_ok(self):
+        raised = False
+        try:
+            self.robot_commander.set_max_acceleration_scaling_factor(0.2)
+        except Exception:
+            raised = True
+        self.assertFalse(raised)
+
+    def test_set_max_acceleration_scaling_factor__range_not_ok(self):
+        raised = False
+        try:
+            self.robot_commander.set_max_acceleration_scaling_factor(3)
+        except Exception:
+            raised = True
+        self.assertTrue(raised)
+
+    def test_allow_looking(self):
+        self.robot_commander.allow_looking(True)
+
+    def test_allow_replanning(self):
+        self.robot_commander.allow_replanning(True)
+
+    def test_plan_to_joint_value_target(self):
+        arm_home_joints_goal = {'ra_shoulder_pan_joint': 0.00, 'ra_elbow_joint': 2.00,
+                                'ra_shoulder_lift_joint': -1.25, 'ra_wrist_1_joint': -0.733,
+                                'ra_wrist_2_joint': 1.5708, 'ra_wrist_3_joint': 0.00}
+        self.robot_commander.plan_to_joint_value_target(arm_home_joints_goal,
+                                                        angle_degrees=False, custom_start_state=None)
+        self.assertTrue(type(self.robot_commander._SrRobotCommander__plan) == RobotTrajectory)
+
+    def test_execute(self):
+        arm_home_joints_goal = {'ra_shoulder_pan_joint': 0.00, 'ra_elbow_joint': 2.00,
+                                'ra_shoulder_lift_joint': -1.25, 'ra_wrist_1_joint': -0.733,
+                                'ra_wrist_2_joint': 1.5708, 'ra_wrist_3_joint': 0.00}
+        self.robot_commander.plan_to_joint_value_target(arm_home_joints_goal,
+                                                        angle_degrees=False, custom_start_state=None)
+        self.robot_commander.execute()
+        executed_joints = self.robot_commander.get_current_state()
+        diffs = [arm_home_joints_goal[key] - executed_joints[key] for key in arm_home_joints_goal
+                 if key in executed_joints]
+        condition_1 = all(diff < 0.05 for diff in diffs)
+        condition_2 = self.robot_commander._SrRobotCommander__plan is None
+        self.assertTrue(condition_1 and condition_2)
+
+    def test_execute_plan(self):
+        arm_home_joints_goal = {'ra_shoulder_pan_joint': 0.00, 'ra_elbow_joint': 2.00,
```

Comment:
if we use always the same joint angles maybe we should reset the arm position for every test, maybe you can have a common function that plans and executes to home position. OR you can use different positions.